asyncapi/sfdc/service_types.bal

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Salesforce event service object.
+public type StreamingEventService service object {
+    # Triggers on a new record create event.
+    # 
+    # + info - The information about the triggered event
+    # + return - `()` on success else an `error`
+    remote function onCreate(EventData info) returns error?;
```

Comment:
Shall we change the parameter name 'info' to payload?. Need to do this in all the remote functions

Reply:
Fixed in f61ad2481ac36a3ca623151376fc771e064d4bae Thanks